webui/cypress/credentials.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ // These are the account credentials used to run e2e test.
+ // Feel free to change it to any values to suit your need.
+ // The test will create them at the start of the test run,
+ // and clean up them at the end.
+
+exports.adminEmail = "frontend-e2e-cwa-admin@gmail.com";
+exports.adminPassword = "crcwae2eadmin";
+
+exports.workerEmail = "frontend-e2e-cwa-worker@gmail.com";
+exports.workerPassword = "crcwae2eworker";
```

Comment:
Unfortunately I don't think we can include these default credentials if they are used for anything in the live application -- it would not pass security review. Would it be possible to instead generate random credentials at runtime?

Reply:
Changed it to auto generate password.